src/vmspawn/vmspawn-util.c

```diff
@@ -33,6 +33,22 @@ OvmfConfig* ovmf_config_free(OvmfConfig *config) {
         return mfree(config);
 }
 
+int parse_qemu_network_stack(const char *s, QemuNetworkStack *net_stack) {
```

Comment:
Just make this return `QemuNetworkStack`? `_QEMU_NET_INVALID` can be returned if no match is found

Reply:
Yeah this is a lot nicer, just didn't occur to me at the time :)